bitcoin/src/blockdata/transaction.rs

```diff
@@ -1339,6 +1339,8 @@ impl<'a> Arbitrary<'a> for Transaction {
 
 #[cfg(test)]
 mod tests {
+    use core::str;
```

Comment:
If this import is unused, why is it being added here?

Reply:
This same issue came up before, the test module is not linted in the same way and the original use statement gaves ```warning: unused import: `str` ```.  Moving the use statement to the test module removes the warning.

Reply:
it's used in the test only

Reply:
removing this from the test module doesn't produce a missing import error, so maybe I'm using the wrong feature flags.

Reply:
cd into the `bitcoin` dir - that's how i found it